### PR TITLE
Adding a potential fix for find_package

### PR DESCRIPTION
**Pull request by msoos**
As per https://github.com/msoos/cryptominisat/pull/607

### CODE REVIEW

File: STPConfig.cmake.in [hunk 1]
```diff
@@ -13,6 +13,7 @@ set(STP_INCLUDE_DIRS "@CONF_INCLUDE_DIRS@")
 
 # Our library dependencies (contains definitions for IMPORTED targets)
 include("${STP_CMAKE_DIR}/@STP_TARGETS_FILENAME@")
+find_package(cryptominisat5 CONFIG)
```
Comment (arrowd) [inline STPConfig.cmake.in]: Not sure if that matters, but how about including it before that `include` statement?

**Comment by msoos** [comment]
Hi All! This has been requested by @arrowd and I'm wondering if you agree -- I feel like my patch it not perfect and may not reflect what his/her wishes were. So Iet's discuss. Basically, we need something like this or the build for KLEE fails and potentially this means that STP build fails when it's part of a larger build.

File: STPConfig.cmake.in (hunk 1 above)
Comment (msoos) [reply STPConfig.cmake.in]: Fixed now :)

**Comment by arrowd** [comment]
U just stumbled upon https://cmake.org/cmake/help/v3.17/module/CMakeFindDependencyMacro.html

I suspect, this is the intended way to specify a dependency in the package file.

**Comment by msoos** [comment]
Cool. Can you please check if it works? Thanks.

**Comment by arrowd** [comment]
Yep, it does. The full patch I used:

```
--- STPConfig.cmake.in.orig	2019-03-02 14:26:01 UTC
+++ STPConfig.cmake.in
@@ -11,6 +11,9 @@
 get_filename_component(STP_CMAKE_DIR "${CMAKE_CURRENT_LIST_FILE}" PATH)
 set(STP_INCLUDE_DIRS "@CONF_INCLUDE_DIRS@")
 
+include(CMakeFindDependencyMacro)
+find_dependency(cryptominisat5)
+
 # Our library dependencies (contains definitions for IMPORTED targets)
 include("${STP_CMAKE_DIR}/@STP_TARGETS_FILENAME@")
```

**Comment by msoos** [comment]
Nice! If you could make that into a pull request, I'll pull that in :) Thanks! In the meanwhile, I am closing this pull request. Looking forward to merging the PR!

**Pull request closed without merging**